lucene/core/src/java/org/apache/lucene/index/MergePolicy.java

```diff
@@ -526,6 +526,19 @@ public abstract MergeSpecification findForcedMerges(
   public abstract MergeSpecification findForcedDeletesMerges(
       SegmentInfos segmentInfos, MergeContext mergeContext) throws IOException;
 
+  /**
+   * Identifies merges that we want to execute (synchronously) on commit. By default, do not synchronously merge on commit.
+   *
+   * Implementers of this method should use isMergingSegment to exclude any already-merging segments from the returned
```

Comment:
What is `isMergingSegment` here?

Reply:
Oops... that's from an earlier implementation, before I realized that `MergeContext` provides information on merging segments.